src/feature_filter/index.ts

```diff
@@ -76,9 +76,10 @@ const filterSpec = {
  *
  * @private
  * @param {Array} filter MapLibre filter
+ * @param {Object} [globalState] - Global state object to be used for evaluating 'global-state' expressions
```

Comment:
I think that those annotations were removed from most of the code (the "{type}"), feel free to remove it from here as well.